learningequality/studio
# Run migrations to fix incorrect source fields of contentnodes

## Summary
We filter out all the contentnodes whose source fields have been changed after being imported and reset them to their original values.

### Manual verification steps performed
1. Ran the query in hotfixes 
2. Verified the returned nodes conform to expected disparities. 

## Reviewer guidance
### How can a reviewer test these changes?
Recommended to run the query in hotfixes and verify the returned nodes. some of the fields to look at would be 
last date of modification and verifying what source_node_fields have been changed. 

### Are there any risky areas that deserve extra testing?
If there is something that is missed migrations may be incorrect and may effect large number of nodes. 

## References
closes #4190 

## Comments


----

### Contributor's Checklist


PR process:


Studio-specifc:



Testing:

___

### Reviewer's Checklist
#### This section is for reviewers to fill out.
